app/controllers/api/v1/results_controller.rb

```diff
@@ -92,10 +130,15 @@ def result_params
       # Partially implement sideposting draft
       # https://github.com/json-api/json-api/pull/1197
       def result_text_params
-        prms =
-          params[:included]&.select { |el| el[:type] == 'result_texts' }&.first
-        prms.require(:attributes).require(:text)
-        prms[:attributes]
+        prms = params[:included]&.select { |el| el[:type] == 'result_texts' }&.first
```

Comment:
Can you simplify these blocks?

Reply:
And make sure that appropriate exception is frown in case of corrupted attributes

Reply:
Isn't this already doing what it should? Do you have any ideas about simplifying this?